scripts/merge-and-build.sh

```diff
@@ -114,6 +139,8 @@ else
   fi
 fi # End check if we are master
 
+# TODO: Origin already vendors the web console. Should we stop updating it separately for OSE?
```

Comment:
Looks like it may make sense to just remove entirely the last commit for web console and re-create it with the below code, but do it in the right order.